### DISCUSSION TITLE
Move lines up/down

### DISCUSSION THREAD

**Discussion by meteorquake** [WinMerge/winmerge#1687] [Ideas]
Most program editors have a shortcut to move the current line(s) (covered by the selection) up or down, such as Alt-ArrowUp/Alt-ArrowDown
It would be good to have such a thing on winmerge, as in many cases of comparison you also want to remove gross differences of position where you've moved a bit of code in order to focus on finer differences.
I realise there's a block-moved detection but in fact you often want to see the full differences initially and examine them one-by-one manually.
I don't think this is already present?
Cheers, David
David

**Comment by sdottaka** [comment]
Unfortunately, there is no such feature in WinMerge.
However, there is a feature called a Synchronization Point, as shown in the gif animation below, which matches a specified left line to a specified right line.
Perhaps this feature will do some of what you want to do.

![#1687](https://user-images.githubusercontent.com/98126/214568093-8971dcfc-fdca-4245-bc88-7ba2d6db5128.gif)
